app/models/category.rb

```diff
@@ -2,4 +2,20 @@ class Category < ActiveRecord::Base
   has_many :item_categories
   has_many :items, through: :item_categories
   validates :name, presence: true, uniqueness: true, length: { maximum: 42 }
+
+  before_save :downcase_name
+
+  def to_param
+    "#{name.parameterize}"
```

Comment:
This is a fantastic method find, I'm excited to see names in our url instead of ids